Code/Framework/AzFramework/AzFramework/Asset/AssetSystemBus.h

```diff
@@ -325,6 +325,8 @@ namespace AzFramework
             virtual void AssetSystemAvailable() {}
             //! Notifies listeners Asset System turns not available
             virtual void AssetSystemUnavailable() {}
+            //! Notifies listeners Asset System is still waiting
+            virtual void AssetSystemWaiting() {}
```

Comment:
As a suggestion, it would be helpful to call out that this message gets called on a frequent periodic basis:

```suggestion
            //! Notifies listeners Asset System is still waiting.
            //! This will get called every N milliseconds during the Asset System startup
            //! until the Asset System becomes available.
            virtual void AssetSystemWaiting() {}
```